2026/using-components-2/slides.md

```diff
@@ -1,13 +1,13 @@
 ---
 titleTemplate: '%s'
-author: Omar Kawach, Max Patiiuk
+author: Omar Kawach, Max Patiiuk, Nicholas Romano
 mdc: true
 colorSchema: dark
 ---
 
-## ArcGIS Maps SDK for JavaScript:<br>App Development with Components<br>part 2: Using Frameworks
+## ArcGIS Maps SDK for JavaScript: App Development with Components,<br>Part 2: Using Frameworks
 
-Omar Kawach, Max Patiiuk, Nick Romano
+Omar Kawach, Max Patiiuk, Nicholas Romano
```

Comment:
@nick-romano I used your name as it appears in Esri Events app so that people can find you easier
feel free to update this slide if you change your name there